theme/p2/base.p2

```diff
@@ -1,12 +1,13 @@
+<!DOCTYPE html>
 <html>
   <head>
     <title>{% block title %}Yurt Tools{% endblock %}</title>
     <link href="/static/css/reset.css" rel="stylesheet">
     <link href="/static/css/theme.css" rel="stylesheet">
   </head>
   <body>
-    <div id="top-bar" class="bar">
-      <div class="bar-item">Yurt Tools</div>
+    <div class="nav">
+      <h1>Yurt Tools</h1>
```

Comment:
Lets make this more specific.  How about `Task Info` and we can eventually have a footer somewhere to put yurt-tools in as a pointer to here.